backend/pennclubs/settings/development.py

```diff
@@ -32,3 +32,13 @@
 
 # Caching settings
 CACHES = {"default": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"}}
+
+# Cybersource settings
+CYBERSOURCE_CONFIG = {
```

Comment:
What is this for?

Reply:
Test config for the Cybersource API, those credentials are taken directly from their API docs/examples